src/features/dotnetTest.ts

```diff
@@ -70,9 +70,7 @@ export function runDotnetTest(testMethod: string, fileName: string, testFramewor
         });
 }
 
-function createLaunchConfiguration(program: string, argsString: string, cwd: string, debuggerEventsPipeName: string) {
-    let args = utils.splitCommandLineArgs(argsString);
```

Comment:
Remove createLaunchConfiguration function or remove the splitCommandLineArgs function?